### DISCUSSION TITLE
Chocolatey support

### DISCUSSION THREAD

**Discussion by TarunavBA** [chocolatey/choco#2492] [Q&A] [answered]
Does chocolatey support windows 7?

**Comment by TarunavBA** [comment]
It currently doesn't support I guess I get a lot of errors spat at me if I run the commands as said in the chocolatey website.

**Reply by TheCakeIsNaOH** [reply]
Unless you post the errors, this is not helpful in figuring out what the issue is.

**Comment by TheCakeIsNaOH** [comment, marked as answer]
Chocolatey does support Windows 7.
However, the chocolatey.org website requires TLS v2 at a minimum, so an unpatched installation of Windows 7 does not work for the standard installation method of Chocolatey (that is, installation directly from the website).

The requirements for TLS v2 support are Powershell v3 or higher, and .Net framework 4.5 or higher. 

Another option would be to use an alternate install method, detailed here:
https://docs.chocolatey.org/en-us/choco/setup#more-install-options

**Reply by gep13** [reply]
There is also some more information about this here: https://blog.chocolatey.org/2020/01/remove-support-for-old-tls-versions/
